src/components/main/Summary.tsx

```diff
@@ -0,0 +1,55 @@
+import Card from "../UI/Card"
+import axios from 'axios'
+import { useQuery } from '@tanstack/react-query'
+import { motion } from 'framer-motion'
+
+const Summary = () => {
+    const fetchDescription = async () => {
+        const res = await axios.get('/.netlify/functions/generate-description')
+        return res.data
+    }
+
+    const response = useQuery({
+        queryKey: ['description'],
+        queryFn: fetchDescription,
+        refetchOnMount: false,
+        refetchOnWindowFocus: false,
+    })
+
+    if (response.status === 'error') console.log(response)
```

Comment:
or the error

Reply:
or loading

Reply:
const { description, isLoading, error } = useDaVinciDescription()

and use the variables in the render